app/editor/src/features/admin/reports/ReportFormInstances.tsx

```diff
@@ -22,7 +22,7 @@ export const ReportFormInstance: React.FC = () => {
 
   React.useEffect(() => {
     findInstancesForReportId(values.id).then((instances) => {
-      setInstances(instances);
+      setInstances(instances ?? []);
```

Comment:
This was resulting in null values when the API had an OOM error for some reason.